parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/BaseRule.g4

```diff
@@ -1984,3 +1985,13 @@ xmlTableOptions
 xmlTableColumn
     : columnName (FOR ORDINALITY | (dataType | XMLTYPE (LP_ SEQUENCE RP_ BY REF)?) (PATH STRING_)? (DEFAULT expr)?)
     ;
+
+multisetExpr
+    : columnName MULTISET multisetOperator (ALL | DISTINCT)? columnName
+    ;
+
+multisetOperator
+    : EXCEPT
+    | INTERSECT
+    | UNION
+    ;
```

Comment:
Hi @lancelly , please keep a blank line at the end of file.

Reply:
Done